sktree/tree/tests/test_tree.py

```diff
@@ -44,3 +60,20 @@ def test_unsupervisedtree():
 
     # a single decision tree does not fit well, but should still have a positive score
     assert score > 0.05
+
+def test_iris():
+    # Check consistency on dataset iris.
+    for (name, Tree), criterion in product(CLF_TREES.items(), CLF_CRITERIONS):
```

Comment:
Use two instances of `pytest.mark.parametrize` instead will make it so that each of these is inherently a separate unit-test.

Reply:
Thank you for the example! I saw this edit after I made the change. Does criterion need to be parameterized too as it's just string?